crates/dojo-world/src/manifest.rs

```diff
@@ -31,7 +31,7 @@ pub const EXECUTOR_CONTRACT_NAME: &str = "dojo::executor::executor";
 pub const BASE_CONTRACT_NAME: &str = "dojo::base::base";
 
 #[derive(Error, Debug)]
-pub enum ManifestError {
+pub enum WorldError {
```

Comment:
`WorldManifestError` may be better, or actually `ManifestError` could be a good thing to keep to cleary identify it's something related to manifests?